process.go

```diff
@@ -78,12 +58,12 @@ func parseTimestamp(str string) int64 {
 		return fallback
 	}
 
-	time, err := dateparse.ParseLocal(str)
+	localTime, err := dateparse.ParseLocal(str)
```

Comment:
Was getting a warning on my IDE (IntelliJ), that says variable named "time" might be troublesome if we're about to import Go's stdlib time package.